exam_12.js

```diff
@@ -0,0 +1,74 @@
+// ex 1: 
+const checkArrayComparison = (arr1 = [], arr2 = []) => {
+  arr1.sort()
+  arr2.sort()
+  if (arr1.length == arr2.length && JSON.stringify(arr1) === JSON.stringify(arr2)) return true
+  else return false
+}
+
+// method2 - chưa hoàn thành code.
+/* 1.tạo hàm inputAray(arr) riêng -> return all=[arr1,arr2,arr3...] 
+2.khởi tạo hàm checkArrayComparison(all)
+3. tạo vòng for sắp xếp tuwnngf array trong all với sort()
+4. xét all[i] và all[j] vs (j =i+1) với điều kiện 
+if (all[i].length == all[j].length && JSON.stringify(all[i]) === JSON.stringify(all[j])
+và đưa ra kết quả true /false*/
+
+// ex 2:
+const result = []
+const flattenArray = (arr = []) => {
```

Comment:
```suggestion
const flattenArray = (arr = []) => {
  const result = []
```

Đưa `result` vào thân hàm, ko được đặt ngoài, biến `result` chỉ nên tồn tại trong hàm đấy thôi